### PR TITLE
trying to fix randomess tutorial on latest template

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: v3/how-to-guides/02-pallet-design/c-randomness/index.mdx [hunk 1]
```diff
@@ -80,7 +80,7 @@ Use a nonce to serve as a subject for the `frame_support::traits::Randomness::ra
 
       // Random value.
       let nonce = Self::get_and_increment_nonce();
-      let randomValue = T::MyRandomness::random(&nonce);
+      let (randomValue, _) = T::MyRandomness::random(&nonce);
```
Comment (sacha-l) [inline v3/how-to-guides/02-pallet-design/c-randomness/index.mdx:83]: Hi @dzmitry-lahoda - why is this necessary ? 🤔

**Review by sacha-l — COMMENTED** [review]
Thanks for surfacing this @dzmitry-lahoda. Just left a question. Cheers!

File: v3/how-to-guides/02-pallet-design/c-randomness/index.mdx (hunk 1 above)
Comment (dzmitry-lahoda) [reply v3/how-to-guides/02-pallet-design/c-randomness/index.mdx:83]: why i split random value out of tuple?

Comment (sacha-l) [reply v3/how-to-guides/02-pallet-design/c-randomness/index.mdx:83]: Yes. I'm assuming it's because it takes `<Hash, BlockNumber>` ? But I'm not sure it's necessary.

**Review by sacha-l — APPROVED** [review]
